osu.Game/Overlays/Mods/DifficultyMultiplierDisplay.cs

```diff
@@ -0,0 +1,185 @@
+                                        {
+                                            Anchor = Anchor.Centre,
+                                            Origin = Anchor.Centre,
+                                            Margin = new MarginPadding { Horizontal = 18 },
+                                            Shear = new Vector2(-ModPanel.SHEAR_X, 0),
+                                            Text = "Difficulty Multiplier",
+                                            Font = OsuFont.Default.With(size: 17, weight: FontWeight.SemiBold)
+                                        }
+                                    }
+                                },
+                                multiplierFlow = new FillFlowContainer
+                                {
+                                    AutoSizeAxes = Axes.Both,
+                                    Anchor = Anchor.Centre,
+                                    Origin = Anchor.Centre,
+                                    Shear = new Vector2(-ModPanel.SHEAR_X, 0),
+                                    Direction = FillDirection.Horizontal,
+                                    Spacing = new Vector2(2, 0),
+                                    Children = new Drawable[]
+                                    {
+                                        multiplierCounter = new MultiplierCounter
+                                        {
+                                            Anchor = Anchor.CentreLeft,
+                                            Origin = Anchor.CentreLeft,
+                                            Current = { BindTarget = Current }
+                                        },
+                                        new SpriteIcon
+                                        {
+                                            Anchor = Anchor.CentreLeft,
+                                            Origin = Anchor.CentreLeft,
+                                            Icon = FontAwesome.Solid.Times,
+                                            Size = new Vector2(7),
+                                            Margin = new MarginPadding { Top = 1 }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+        }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            contentBackground.Colour = colourProvider.Background4;
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            current.BindValueChanged(_ => updateState(), true);
+            FinishTransforms(true);
+            // required to prevent the counter initially rolling up from 0 to 1
+            // due to `Current.Value` having a nonstandard default value of 1.
+            multiplierCounter.SetCountWithoutRolling(Current.Value);
```

Comment:
Weird this isn't handled by the `FinishTransforms` above, but it does look required. Probably not worth dwelling on for now.